fabric/utils.py

```diff
@@ -325,9 +325,11 @@ def error(message, func=None, exception=None, stdout=None, stderr=None):
     import fabric.state
     if func is None:
         func = fabric.state.env.warn_only and warn or abort
-    # If debug printing is on, append a traceback to the message
-    if fabric.state.output.debug:
-        message += "\n\n" + format_exc()
+    # If exception printing is on, append a traceback to the message
+    if fabric.state.output.exceptions:
```

Comment:
Let's preserve backwards compatibility by making this line say `if fabric.state.output.exceptions or fabric.state.output.debug` :)